### PR TITLE
Create feedback

### LINKED ISSUE #45: Instructor Create Feedback Endpoint
- [x] `POST /api/v1/instructors/:instructor_id/students/:student_id/student_projects`

Required body params:
- [x] instructor_id
- [x] student_id
- [x] project_template_id
- [x] rubric_template_category_id
- [x] score

Optional:
- [x] comment

### PR DESCRIPTION
**Pull request by Gvieve**
### Add
* create student_project and feedback endpoint
* method on ProjectFeedback model to create records from post request
### Update / Remove
* Projects poro to send single project that has been created as response

Close #45

### CODE REVIEW

File: app/controllers/api/v1/student_projects_controller.rb [hunk 1]
```diff
@@ -16,6 +16,19 @@ def update
     render_success(ProjectsSerializer.new(projects))
   end
 
+  def create
```
Comment (jequeza) [inline app/controllers/api/v1/student_projects_controller.rb:19]: yay Feedback

File: app/controllers/api/v1/student_projects_controller.rb [hunk 2]
```diff
@@ -16,6 +16,19 @@ def update
...
+    return render_error("Required parameter missing") if missing_params(required_create)
+    project = StudentProject.new(student_project_params) if student && instructor
+    if project.save
+      ProjectFeedback.create_feedback(project, params) if !params[:project_feedback].nil?
+      params[:mod] = project.project_template.mod
+      params[:id] = project.id
+      projects = Projects.new(params)
+      render_success(ProjectsSerializer.new(projects))
+    else
```
Comment (b-enji-cmd) [inline app/controllers/api/v1/student_projects_controller.rb:28]: Wondering if there's a render failure here?

**Review by jequeza — APPROVED** [review]
Looks awesome Dev! Great Job!

**Review by b-enji-cmd — APPROVED** [review]
LGTM! Awesome work!!!!!

File: app/controllers/api/v1/student_projects_controller.rb (hunk 2 above)
Comment (Gvieve) [reply app/controllers/api/v1/student_projects_controller.rb:28]: I will add that in for sad path next. Great q!

**Pull request merged**